src/pages/Home.jsx

```diff
@@ -1,7 +1,30 @@
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/react';
-import React from 'react';
+import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar, IonItem, IonLabel, IonList } from '@ionic/react';
+import React,{useEffect, useState} from 'react';
+import axios from 'axios';
 
 const Home = () => {
+
+  const [articles, setArticles] = useState ({})
+  let articleItems
+
+  useEffect(() => {
```

Comment:
swap the place of this and the getArticles function for easier readability

Reply:
ok